src/main.js

```diff
@@ -4,9 +4,10 @@ var homeButton = document.querySelector('.home-button');
 var randomizeButton = document.querySelector('.random-cover-button');
 var saveCoverButton = document.querySelector('.save-cover-button');
 var viewSavedButton = document.querySelector('.view-saved-button');
-var makeNewButton = document.querySelector('.make-new-button')
+var userCoverButton = document.querySelector('.make-new-button')
```

Comment:
I think this is a good change, and that we should consider renaming our button variables and functions with a clearer naming convention. (But not for this iteration.)

Reply:
cool let's do that next time we meet and we can go over it.